scripts/deploy-staking-token.js

```diff
@@ -63,32 +63,32 @@ async function main() {
     let runtimeData = { abi, address };
     fs.writeFileSync(path.join(__dirname, '../parity-data/StakingToken.json'), JSON.stringify(runtimeData, null, 4));
 
-    console.log('**** Set ValidatorSetContract address in StakingToken contract');
+    console.log('**** Set StakingAuRa address in StakingToken contract');
     let tx1 = await SnS(web3, {
         from: OWNER,
         to: address,
-        method: StakingTokenInstance.methods.setValidatorSetContract(ValidatorSetContract.address),
+        method: StakingTokenInstance.methods.setStakingContract(StakingAuRa.address),
         gasPrice: '0',
     });
     pp.tx(tx1);
     expect(tx1.status).to.equal(true);
 
-    console.log('**** Set StakingToken address in ValidatorSetContract');
+    console.log('**** Set StakingToken address in StakingAuRa: ' + StakingAuRa);
```

Comment:
`+ StakingAuRa);` part just prints `[object Object]`, so probably is not necessary.